src/browser/extension/utils/filters.js

```diff
@@ -1,5 +1,11 @@
 import mapValues from 'lodash/mapValues';
 
+export const FilterState = {
```

Comment:
The filter states are used in different parts of the code, so I made an exported enum with them.